boards/lilygo/ttgo_t7v1_5/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Tobias Kässer <t.kaesser@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_TTGO_T7V1_5
+	select SOC_ESP32
+
+config ESP_WIFI_MAX_THREAD_PRIORITY
+    int "ESP WiFi Maximum Thread Priority"
+    default 3
```

Comment:
Don't know what this is but it can go

Reply:
Without it i'm getting these kind of errors on build:
```
/home/cite/emb-test/projects/lora-receiver-zephyr/prj.conf:5: warning: attempt to assign the value '6' to the undefined symbol ESP_WIFI_MAX_THREAD_PRIORITY
```